spec/components/candidate_interface/rejections_component_spec.rb

```diff
@@ -57,6 +58,33 @@
     end
   end
 
+  describe "when the rejection reason type is 'enic rejection_reasons'" do
+    let(:application_form) { create(:application_form, :minimum_info, first_nationality: 'Canadian', right_to_work_or_study: 'no') }
+    let(:application_choice) { build_stubbed(:application_choice, :with_structured_rejection_reasons, application_form: application_form) }
+    let!(:application_qualification) { create(:degree_qualification, award_year: '2014', enic_reference: nil, institution_country: 'FR', application_form: application_form) }
+
+    it 'renders using RejectionReasonsComponent' do
+      result = render_inline(described_class.new(application_choice:))
+      expect(result.text).to include('Qualifications')
+      expect(result.text).to include('No maths GCSE at minimum grade 4 or C, or equivalent')
+      expect(result.text).to include('Could not verify qualifications:')
+      expect(result.text).to include('We could find no record of your GCSEs.')
+      expect(result.text).to include('Personal statement')
+      expect(result.text).to include('Quality of writing:')
+      expect(result.text).to include('We do not accept applications written in Old Norse.')
+    end
+
+    it 'renders a link to Apply for a statement of comparability with no enic_reference, a non uk qualification and could not verify qualifications rejection reason' do
+      course = build_stubbed(:course)
+      allow(application_choice).to receive_messages(course: course)
+
+      result = render_inline(described_class.new(application_choice:, render_link_to_find_when_rejected_on_qualifications: true))
+      expect(result.text).to include('If you decide to apply again, think about including a statement of comparability from UK ENIC')
+      expect(result.css('.govuk-link')[0][:href]).to eq('https://www.enic.org.uk/Qualifications/SOC/Default.aspx')
```

Comment:
Is there a preference for finding the CSS or could we use [`have_link`](https://capybara-test-helpers.netlify.app/api/#has-link) to simplify this?